tests/lsp/location_compiler_test_runner_base.toit

```diff
@@ -8,6 +8,11 @@ import reader show BufferedReader
 import .lsp_client show LspClient run_client_test
 import .utils
 
+is_absolute_ path/string -> bool:
+  if platform == PLATFORM_WINDOWS:
+    return path.size > 1 and path[1] == ':'
```

Comment:
I wonder if sometimes we see unix-style paths on Windows too?
Perhaps Windows should have both tests.

Reply:
This is only for our test, where we control the inputs (CMake), but done.